src/Stratis.Bitcoin/Connection/PeerBanning.cs

```diff
@@ -98,7 +104,8 @@ public void BanAndDisconnectPeer(IPEndPoint endpoint, int banTimeSeconds, string
             foreach (var peerAddress in peerAddresses)
             {
                 peerAddress.BanTimeStamp = this.dateTimeProvider.GetUtcNow();
-                peerAddress.BanUntil = this.dateTimeProvider.GetUtcNow().AddSeconds(banTimeSeconds);
+                peerAddress.BanUntil = this.dateTimeProvider.GetUtcNow().AddSeconds(
```

Comment:
If I remember correctly, we accept a zero second ban when we just want to disconnect a peer. I will try to find the issue where this last came up, but for now I'm not sure this logic is desirable.

Reply:
Look at issue https://github.com/stratisproject/StratisBitcoinFullNode/issues/2815